package.json

```diff
@@ -50,23 +50,25 @@
     "@glimmer/component": "1.1.2",
     "@glimmer/tracking": "1.1.2",
     "@html-next/vertical-collection": "4.0.0",
+    "@types/node": "*",
```

Comment:
Please elaborate

Reply:
I got the following warning:
```bash
 WARN  Issues with peer dependencies found
.
└─┬ @adfinis-sygroup/semantic-release-config 3.4.0
  └─┬ @commitlint/cli 16.0.2
    └─┬ @commitlint/load 16.0.0
      └─┬ cosmiconfig-typescript-loader 1.0.3
        ├── ✕ missing peer @types/node@"*"
        └─┬ ts-node 10.9.1
          └── ✕ missing peer @types/node@"*"
Peer dependencies that should be installed:
  @types/node@"*"
```
Did not spent much though, just wanted to

Reply:
Same as above